.travis.yml

```diff
@@ -24,6 +24,8 @@ matrix:
       env: DEPS=dev SYMFONY_VERSION='3.4.*'
     - php: 7.1
       env: DEVS=dev SYMFONY_VERSION='4.0.*'
+    - php: 7.2
```

Comment:
I would suggest add a PHP 7.3 and 7.4 build as well

Reply:
Sounds logical to me, although, PHP 7.4 may generate some unexpected errors due it's a very recent version, but I'm just guessing